### ISSUE TITLE
dist-server is duplicated in the .gitignore

### ISSUE THREAD

**Issue by Yura13** [angular/angular-cli#10063]
### Versions
Angular CLI: 1.7.3
Node: 8.10.0
OS: win32 x64
Angular: 5.2.9

### Repro steps
Folder name 'dist-server' present in .gitignore after ng new projectName.
After run ng g universal serveApp 'dist-server' again added at the end of the .gitignore file.

### Observed behavior
Folder name 'dist-server' is duplicated in the .gitignore file.

### Desired behavior
Folder name 'dist-server' should add only once in the .gitignore file.

**Comment by alan-agius4** [comment]
Thanks for reporting this issue. This issue is now obsolete due to changes in the recent releases. Please update to the most recent Angular CLI version.

If the problem persists after upgrading, please open a new issue, provide a simple repository reproducing the problem, and describe the difference between the expected and current behavior.
